pkg/cmd/bootstrap_test.go

```diff
@@ -233,14 +227,14 @@ func TestDependenciesWithNothingInstalled(t *testing.T) {
 	fakeClient := newFakeClient(nil, nil)
 
 	wantMsg := `
-Checking if Sealed Secrets is installed with the default configuration[Please install Sealed Secrets operator from OperatorHub]
-Checking if ArgoCD Operator is installed with the default configuration[Please install ArgoCD operator from OperatorHub, with an ArgoCD resource called 'argocd']
-Checking if OpenShift Pipelines Operator is installed with the default configuration[Please install OpenShift Pipelines operator from OperatorHub]`
+Checking if Sealed Secrets is installed with the default configuration [Please install Sealed Secrets operator from OperatorHub]
+Checking if ArgoCD Operator is installed with the default configuration [Please install ArgoCD operator from OperatorHub]
+Checking if OpenShift Pipelines Operator is installed with the default configuration [Please install OpenShift Pipelines operator from OperatorHub]`
 
 	buff := &bytes.Buffer{}
 	fakeSpinner := &mockSpinner{writer: buff}
-	err := checkBootstrapDependencies(&BootstrapParameters{&pipelines.BootstrapOptions{}}, fakeClient, fakeSpinner)
-	wantErr := "Failed to satisfy the required dependencies"
+	_, err := checkBootstrapDependencies(&BootstrapParameters{&pipelines.BootstrapOptions{}}, fakeClient, fakeSpinner)
```

Comment:
None of the unit tests validate the returned dependencies values.  Please add checks.